toxav/ring_buffer.h

```diff
@@ -0,0 +1,18 @@
+#ifndef RING_BUFFER_H
```

Comment:
This header needs a comment as to the point of RingBuffer (and maybe a rename!), 

FYI ring buffer stores AV packets received so ToxAV can calculate the mean on how many packets are lost/received.